src/main/resources/application.properties

```diff
@@ -53,7 +53,23 @@ playlist.redis.key.mapping={"ORG_FEATURED_COURSES":"orgID_Featured_Courses"}
 playlist.redis.ttl=86400
 default.content.properties=name,identifier,competencies_v5,avgRating,primaryCategory,contentType,channel,organisation,createdFor,duration,creatorLogo,appIcon,posterImage,additionalTags,description,status,secureSettings
 #content-service
-content-service-host=http://192.168.1.7:6004/
+content-service-host=http://localhost:6004/
 content-read-endpoint=content/v4/admin/read
 content-read-endpoint-fields=?fields=primaryCategory,identifier,batches,versionKey,additionalTags
-redis.insights.index=12
\ No newline at end of file
+redis.insights.index=12
+
+spring.kafka.bootstrap.servers=localhost:9092
+kakfa.offset.reset.value=latest
+kafka.max.poll.interval.ms=15000
+kafka.max.poll.records=100
+kafka.auto.commit.interval.ms=10000
+kafka.topic.demand.request=dev.demand.request
+kafka.topic.demand.content.group="demand.request.group"
+
+notify.service.host=http://localhost:6006
```

Comment:
just mention that notificationServiceIP -- so that we can add an alias in the /etc/hosts and configure it once -- instead of multiple times...